NetworkOfPrivateClinics/NetworkOfPrivateClinics/Program.cs

```diff
@@ -1,33 +1,88 @@
 ﻿// See https://aka.ms/new-console-template for more information
 
-using NetworkOfPrivateClinics;
+using CsvHelper;
+using NetworkOfPrivateClinics.BisinessLogic;
+using NetworkOfPrivateClinics.Interfaces;
+using NetworkOfPrivateClinics.WorkingWithFiles;
+using Newtonsoft.Json;
+using System.Globalization;
+using System.Net.Http.Headers;
+using System.Text.Json.Serialization;
 
 namespace NetworkOfPrivateClinics
 {
-    public static class Program
+    public class Program
     {
-        private static readonly Data generatedInformation;
-        private static readonly ClinicRepository clinicRepository;
+        private static Data generatedInformation;
+        private static ClinicRepository clinicRepository;
+        private static List<Clinic> clinics;
+        //private static string path = """C:\Users\OlehKozlovskyi\Documents\GitHub\NetworkOfClinics\NetworkOfPrivateClinics\source\Oleh.json""";
 
-        static Program() 
+        public Program() 
         {
             generatedInformation = new Data();
             clinicRepository = new ClinicRepository(generatedInformation.Clinics);
+            clinics = new();
         }
 
         public static void Main()
         {
-            Console.WriteLine(">>>>>>>>>>>> MENU <<<<<<<<<<<<".PadLeft(30));
-            Console.WriteLine("1) Get all hospitals".PadLeft(20));
-            Console.WriteLine("2) Get hospital by id".PadLeft(20));
+            CreateStartupMenu();
+            Console.Clear();
+            CreateMenu();
+        }
+
+        public static void CreateStartupMenu()
+        {
+            Console.WriteLine("<<<<<<<<<<<<<<<<< STARTUP MENU >>>>>>>>>>>>>>>");
+            Console.WriteLine("2) Enter directory to source file: ");
+            string path = Console.ReadLine();
+            clinics = new FileReader().Read(path);
+        }
+
+        public static void CreateMenu()
+        {
+            Console.WriteLine(">>>>>>>>>>>> MENU <<<<<<<<<<<<");
+            Console.WriteLine("1) Get all hospitals");
+            Console.WriteLine("2) Get hospital by id");
+            Console.WriteLine("3) Write data to file");
             Console.WriteLine("Select option");
             int optionsNumber = Convert.ToInt32(Console.ReadLine());
             ManageOptions(optionsNumber);
         }
 
+        public static IFileWriter GetInformationFromUser()
+        {
+            Console.Clear();
+            Console.WriteLine("Set directory for saving file: ");
+            string path = Console.ReadLine();
+            Console.WriteLine("Set file name: ");
+            string name = Console.ReadLine();
+            Console.WriteLine("Choose one of the available formats: ");
+            Console.WriteLine("1) CSV ");
+            Console.WriteLine("2) JSON");
+            int choice = int.Parse(Console.ReadLine());
+            return TransformUserData(path, name, choice);
+        }
```

Comment:
1. This method GetInformationFromUser is unclear - what information we should get?
2. Why we return IFileWriter?
3. Why do we need data transformer? If we work with JSON then only json, if we work with csv, then only csv